inference-engine/src/inference_engine/cnn_network_ngraph_impl.cpp

```
@@ -145,6 +158,35 @@ CNNNetworkNGraphImpl::CNNNetworkNGraphImpl(const std::shared_ptr<Function>& nGra
     }
 }
 
+CNNNetworkNGraphImpl::CNNNetworkNGraphImpl(const ICNNNetwork& network) {
+    if (network.getFunction() == nullptr) {
+        cnnNetwork.reset(new CNNNetworkImpl(network));
+        return;
+    }
```

Why do we need this ctor? I expect that in this release can be finally remove `CNNNetworkNGraphImpl` dependency on `CNNNetworkImpl`

---

In general case we can throw exception. I guess it doesn't matter.

---

better to throw an exception I suppose

---

Ok, will change to exception